src/hawkmoth/parser.py

```diff
@@ -31,6 +31,7 @@
 """
```

Comment:
Typo in commit message: 'fro' -> 'from'.

Thanks for the fix! Looks good to me.

Reply:
Thanks, fixed!